src/ImageRequest.js

```diff
@@ -107,17 +107,12 @@ class ImageRequest {
     return true
   }
 
-  /**
-   * Decodes the image request path associated with default
-   * image requests. Provides error handling for invalid or undefined path values.
-   * @param {Object} event - The proxied request object.
-   */
-  _parseQueryParams () {
-    let qp = this.event.queryStringParameters
```

Comment:
The docbock didn't seem to match the fn…and I didn't see you using `_methods` anywhere else in the codebase…